### ISSUE TITLE
how many epoch when training (coco)

### ISSUE THREAD

**Issue by alicera** [ultralytics/ultralytics#852]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

How many epoch do you set to get the coco val AP 52.9 yolov8l?
All of them are 100 epoch?
https://github.com/ultralytics/ultralytics

**Comment by Laughing-q** [comment]
@alicera @triple-Mu yes, it's 500 epochs from scratch. 100 epochs in args file is for training from our pretrained model.

**Comment by LaCandela** [comment]
> @alicera @triple-Mu yes, it's 500 epochs from scratch. 100 epochs in args file is for training from our pretrained model.

And what kind of learning rate schedule did you use? As I see the default is linear schedule but there is an option for cosine schedule. Which one would you recomend?

**Comment by Laughing-q** [comment]
@LaCandela We have a arg `cos_lr` for using cosine schedule and it should work by setting `True`. I recommend using default schedule(linear schedule) first then try another if the metrics doesn't look good to you.
